podium-uaa/src/main/java/nl/thehyve/podium/web/rest/OrganisationResource.java

```diff
@@ -91,21 +100,18 @@ protected static void copyProperties(Organisation source, Organisation target) {
     @SecuredByOrganisation(authorities = AuthorityConstants.ORGANISATION_ADMIN)
     @PutMapping("/organisations")
     @Timed
-    public ResponseEntity<Organisation> updateOrganisation(@OrganisationParameter @Valid @RequestBody Organisation organisation)
-        throws URISyntaxException {
-        log.debug("REST request to update Organisation : {}", organisation);
-        if (organisation.getId() == null) {
-            return createOrganisation(organisation);
-        }
-        Organisation result = organisationService.findOne(organisation.getId());
-        if (result == null) {
-            throw new ResourceNotFound(String.format("Organisation not found with id: %d", organisation.getId()));
-        }
-        copyProperties(organisation, result);
-        organisationService.save(result);
-        return ResponseEntity.ok()
-            .headers(HeaderUtil.createEntityUpdateAlert(ENTITY_NAME, organisation.getId().toString()))
-            .body(result);
+    public ResponseEntity<OrganisationDTO> updateOrganisation(@OrganisationParameter @Valid @RequestBody OrganisationDTO organisationDTO)
```

Comment:
But for this to work, `OrganisationDTO` should implement `IdentifiableOrganisation`. I wonder why the tests pass, I must have missed a case.